cypress/integration/DrugDictionaryAppintegration/DrugDictionaryAppintegration.js

```diff
@@ -0,0 +1,116 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+And('the {string} title is {string}', (tag, title) => {
+    if (title.includes('XX')) {
+        const regex = new RegExp(title.replace(/XX/g, '[0-9]+'));
+        cy.get(`div#NCI-dd-app-root ${tag}`, { timeout: 10000 }).invoke('text').should('match', regex);
+    }
+    else {
+        cy.get(`div#NCI-dd-app-root ${tag}`).should('have.text', title);
+    }
+});
+
+Then('the introductory text appears', () => {
+    cy.get('#NCI-dd-app-root').find('p').should('be.visible').invoke('text').should('not.be.empty');
+});
+
+And('search bar is displayed below the introductory text', () => {
+    cy.get('form[data-testid="tid-search-container"]').should('be.visible');
+    cy.document().then(doc => {
+        expect(doc.querySelector('form[data-testid="tid-search-container"]').parentNode.previousElementSibling.tagName).to.be.eq('P');
+    });
+});
+
+Then('{string} option is displayed below the search bar', (browse) => {
```

Comment:
This step is exact copy of glossary - why copy code when this step already exist ?

Reply:
Drug Dictionary and Glossary has different locators. If I try to use generic locator div.search-box-container, it fails for definition page as definition page has locator div.search-box-container.bottom

Reply:
Glossary folder does not have the beforeEach file